### PR TITLE
Update en.yml to add translations

### LINKED ISSUE #10384: Add translation keys and strings into en.yml
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [en.yml](https://github.com/publiclab/plots2/blob/main/config/locales/en.yml) in the `Plot2 ` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
+ 410 locations:
+ 411    form:
+ 412      scale: "Scale"
+ 413      world: "World"
+ 414      building: "Building"
+ 415      search_name: "Search by place name"
+ 416      coordinates: 'Or <a href="#" id="coord_button">by entering coordinates</a>'
+ 417      latitude: "Latitude"
+ 418      longitude: "Longitude"
+ 419      current_location: "Use current location"
+ 420      blur_location: "Blur my location"
+ 421      learn_more: "Learn more »"
+ 422      location_saved: "How this location will be saved"
+ 423      save: "Save"
+ 424    modal:
+ 425      location: "Enter a location"
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by anayo-hub**
my outreachy task: Add translation keys and strings into en.yml
Fixes #10384

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -407,6 +407,22 @@ en:
...
+      coordinates: 'Or <a href="#" id="coord_button">by entering coordinates</a>'
+      latitude: "Latitude"
+      longitude: "Longitude"
+     current_location: "Use current location"
+      blur_location: "Blur my location"
+     learn_more: "Learn more »"
+    location_saved: "How this location will be saved"
+      save: "Save"
+    modal:
+    location: "Enter a location"
```
Comment (TildaDares) [inline config/locales/en.yml:425]: Hi @anayo-hub, the key-value pairs are not indented properly. Take a look at the issue to see how they should be indented. Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: config/locales/en.yml (hunk 1 above)
Comment (anayo-hub) [reply config/locales/en.yml:425]: should i restart it afresh, to affect the changes @TildaDares

Comment (DalilaBA) [reply config/locales/en.yml:425]: hi @anayo-hub no you just need to modify the file and push the comment to "patch-3" branch on your forked repo, and tell us if you need help 😄

Comment (TildaDares) [reply config/locales/en.yml:425]: @anayo-hub To modify the file as @DalilaBA said, you navigate to this link https://github.com/anayo-hub/plots2/edit/patch-3/config/locales/en.yml?pr=%2Fpubliclab%2Fplots2%2Fpull%2F10462 and indent those lines properly.

Comment (anayo-hub) [reply config/locales/en.yml:425]: @DalilaBA and @TildaDares i have done it, please review

Comment (TildaDares) [reply config/locales/en.yml:425]: Hi @anayo-hub, it's not quite there yet. You just have to follow how the `like` key on line 426 was indented. Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Comment by anayo-hub** [comment]
please @TildaDares review it i have checked it

File: config/locales/en.yml [hunk 2]
```diff
@@ -407,6 +407,22 @@ en:
...
+      coordinates: "Or <a href="#" id="coord_button">by entering coordinates</a>"
+      latitude: "Latitude"
+      longitude: "Longitude"
+      current_location: "Use current location"
+      blur_location: "Blur my location"
+      learn_more: "Learn more »"
+      location_saved: "How this location will be saved"
+      save: "Save"
+  modal:
+    location: "Enter a location"
```
Comment (TildaDares) [inline config/locales/en.yml:425]: @anayo-hub `modal` on line 424 should be indented the same way `form` on line 411 is indented. Line 425 should also be indented.

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: config/locales/en.yml (hunk 2 above)
Comment (anayo-hub) [reply config/locales/en.yml]: @TildaDares modal is done

Comment (anayo-hub) [reply config/locales/en.yml]: @TildaDares pls i'm waiting

File: config/locales/en.yml [hunk 3]
```diff
@@ -407,6 +407,22 @@ en:
...
+      coordinates: "Or <a href="#" id="coord_button">by entering coordinates</a>"
+      latitude: "Latitude"
+      longitude: "Longitude"
+      current_location: "Use current location"
+      blur_location: "Blur my location"
+      learn_more: "Learn more »"
+      location_saved: "How this location will be saved"
+      save: "Save"
+      modal:
+      location: "Enter a location"
```
Comment (TildaDares) [inline config/locales/en.yml:425]: @anayo-hub You also need to indent line 425.

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: config/locales/en.yml (hunk 3 above)
Comment (anayo-hub) [reply config/locales/en.yml]: done @TildaDares

**Comment by anayo-hub** [comment]
@DalilaBA @TildaDares review needed here #10462

File: config/locales/en.yml [hunk 4]
```diff
@@ -407,6 +407,22 @@ en:
...
+      search_name: "Search by place name"
+      coordinates: "Or <a href="#" id="coord_button">by entering coordinates</a>"
+      latitude: "Latitude"
+      longitude: "Longitude"
+      current_location: "Use current location"
+      blur_location: "Blur my location"
+      learn_more: "Learn more »"
+      location_saved: "How this location will be saved"
+      save: "Save"
+      modal:
```
Comment (TildaDares) [inline config/locales/en.yml:424]: @anayo-hub Line 424 needs to be on the same line as line 411

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: config/locales/en.yml [hunk 5]
```diff
@@ -407,6 +407,22 @@ en:
...
+        location: "Enter a location"
```
Comment (TildaDares) [inline config/locales/en.yml:425]: @anayo-hub Line 425 needs to be on the same line as line 423

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: config/locales/en.yml (hunk 5 above)
Comment (anayo-hub) [reply config/locales/en.yml]: done

**Comment by anayo-hub** [comment]
@TildaDares thanks for all the help, kindly review

File: config/locales/en.yml (hunk 5 above)
Comment (anayo-hub) [reply config/locales/en.yml]: doon #10462

File: config/locales/en.yml (hunk 4 above)
Comment (TildaDares) [reply config/locales/en.yml:424]: @anayo-hub Line 424 needs to be indented the same way as line 411

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Comment by anayo-hub** [comment]
all done @TildaDares #10462

**Review by TildaDares — APPROVED** [review]
Thanks @anayo-hub

**Comment by anayo-hub** [comment]
please what's next @TildaDares and thanks for all the help

**Comment by DalilaBA** [comment]
@anayo-hub congratulations !!! well done great wok 😄 🎊 ❤️

**Comment by jywarren** [comment]
Thanks everyone - @anayo-hub and @TildaDares and @DalilaBA for working through this together! I'm merging it now, very much appreciated. 🎉

**Pull request merged**